x-pack/plugins/secops/public/components/drag_and_drop/draggable_wrapper.tsx

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { defaultTo } from 'lodash/fp';
+import * as React from 'react';
+import { Draggable, Droppable } from 'react-beautiful-dnd';
+import { connect } from 'react-redux';
+import styled from 'styled-components';
+import { ActionCreator } from 'typescript-fsa';
+
+import { dragAndDropActions } from '../../store/local/drag_and_drop';
+import { IdToDataProvider } from '../../store/local/drag_and_drop/model';
+import { dataProvidersSelector } from '../../store/local/drag_and_drop/selectors';
+import { State } from '../../store/reducer';
+import { DataProvider, DataProviderId } from '../timeline/data_providers/data_provider'; // tslint:disable-line:no-unused-variable (linter bug!)
+import { getDraggableId, getDroppableId } from './helpers';
+
+const ReactDndDropTarget = styled.div``; // required by react-beautiful-dnd:
```

Comment:
We can't just use a `div` directly because, per the ` // required by react-beautiful-dnd` comment on that line, the `react-beautiful-dnd` API needs a `styled-component` for the `innerRef` when it's used below:
```
<ReactDndDropTarget
  innerRef={droppableProvided.innerRef}
  {...droppableProvided.droppableProps}
>
```

Specifically, a regular `div` does not have the `innerRef` being used above.